### PR TITLE
Glp week4

### PR DESCRIPTION
**Pull request by geena-panzitta**

### CODE REVIEW

File: Geena - geena-panzitta/GLP_week4assignment.do [hunk 1]
```diff
@@ -0,0 +1,289 @@
...
+Q1. a
+*******************************************************************************/
+{
+
+gen pixel_consistent = .
+label variable pixel_consistent "Are payouts consistent without pixel?"
+
+bysort pixel: egen pixel_payout_min = min(payout)
+bysort pixel: egen pixel_payout_max = max(payout)
+label variable pixel_payout_min "Minimum payout within pixel"
```
Comment (BeaLeydier) [inline Geena - geena-panzitta/GLP_week4assignment.do]: This is the wrong variable label

```suggestion
label variable pixel_payout_min "Maximum payout within pixel"
```

File: Geena - geena-panzitta/GLP_week4assignment.do [hunk 2]
```diff
@@ -0,0 +1,289 @@
...
+destring(pixel_numerical), replace
+
+
+gen pixel_village = .
+label variable pixel_village "Is the village within one pixel?"
+
+bysort village: egen pixel_min = min(pixel_numerical)
+bysort village: egen pixel_max = max(pixel_numerical)
+label variable pixel_min "Minimum pixel within village"
+label variable pixel_max "Maximum pixel within village"
```
Comment (BeaLeydier) [inline Geena - geena-panzitta/GLP_week4assignment.do:65]:
```suggestion
bysort village: egen pixelmin = min(pixelnumerical)
bysort village: egen pixelmax = max(pixelnumerical)
label variable pixel_min "Minimum pixel within village"
label variable pixel_max "Maximum pixel within village"
```

File: Geena - geena-panzitta/GLP_week4assignment.do [hunk 3]
```diff
@@ -0,0 +1,289 @@
...
+	local varlist "kiswahili english maarifa hisabati science uraia average"
+	foreach subject in `varlist'{
+		replace `subject' = usubstr(`subject',-1,1)
+	}
+	//end: code from hint_q2
+	append using `allstudents'
+	save `allstudents', replace
+}
+
+use `allstudents', clear
```
Comment (BeaLeydier) [inline Geena - geena-panzitta/GLP_week4assignment.do:151]: this doesn't run

**Review by BeaLeydier — CHANGES_REQUESTED** [review]
See in line comments

File: Geena - geena-panzitta/GLP_week4assignment.do [hunk 4]
```diff
@@ -0,0 +1,289 @@
...
+use village_pixel.dta, clear
+
+}
+/*******************************************************************************
+Q1. a
+*******************************************************************************/
+{
+
+gen pixel_consistent = .
+label variable pixel_consistent "Are payouts consistent without pixel?"
```
Comment (shailyacharya) [inline Geena - geena-panzitta/GLP_week4assignment.do:29]: Nice use of variable labels-- you reminded me that I should probably add this to my code as well!

File: Geena - geena-panzitta/GLP_week4assignment.do (hunk 3 above)
Comment (shailyacharya) [reply Geena - geena-panzitta/GLP_week4assignment.do:151]: I am running into the same issue as Bea!

File: Geena - geena-panzitta/GLP_week4assignment.do [hunk 5]
```diff
@@ -0,0 +1,289 @@
...
+	
+clear
+
+use grant_prop_review_2022.dta, clear
+
+global students ahs103 am3944 ass95 ato12 bp557 fg443 ft227 glp38 mp1792 nbs56 nd549 oo140 sa1600 scb136 yc577 ynd3
+
+rename (Rewiewer1 Review1Score) (Reviewer1 Reviewer1Score)
+
+foreach i in $students {
```
Comment (shailyacharya) [inline Geena - geena-panzitta/GLP_week4assignment.do]: Is there a reason why you defined students as a global, instead of as a local?

**Review by shailyacharya — APPROVED** [review]

**Pull request merged**